docker-build/build_cluster_docker.sh

```diff
@@ -150,6 +150,16 @@ buildModule() {
   cp -r ${package_dir}/examples ${source_dir}/docker-build/docker/modules/python/examples
   cp -r ${package_dir}/fate.env ${source_dir}/docker-build/docker/modules/python/fate.env
 
+  # python-nn
```

Comment:
I think these commands are unnecessary, since we can use the following statements in Dockerfile:
```
From federatedai/python:{version}
```
The base image ` federatedai/python:{version}` already contains all these folders.